OpenRA.Mods.Cnc/UtilityCommands/LegacyTilesetImporter.cs

```diff
@@ -65,8 +65,8 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			var name = args.Length > 3 ? args[3] : Path.GetFileNameWithoutExtension(args[2]);
 			metadata.AppendLine("\tName: {0}".F(name));
 			metadata.AppendLine("\tId: {0}".F(name.ToUpperInvariant()));
-			metadata.AppendLine("\tPalette: iso{0}.pal".F(extension));
-			metadata.AppendLine("\tHeightDebugColors:  00000080, 00004480, 00008880, 0000CC80, 0000FF80, 4400CC80," +
+			metadata.AppendLine("\tExtensions: .{0}, .shp".F(extension));
```

Comment:
Maybe I'm missing something, but there's no `Extensions` field in bleed's `TileSet` code...?

Reply:
Yeah, the fix against bit rot has bit rotted again. Should be fixed now.